.gitignore

```diff
@@ -2,3 +2,4 @@
 **/*.jar
 **/*.zip
 **/*.tar.gz
+.idea/
```

Comment:
@Djelibeybi the .gitignore is so that when we do a git commit it does not include any of the installers that might be in the directory